src/script/api/game/game_window.hpp.sq

```diff
@@ -1023,6 +1023,7 @@ void SQGSWindow_Register(Squirrel *engine)
 	SQGSWindow.DefSQConst(engine, ScriptWindow::WID_GO_RESOLUTION_DROPDOWN,                "WID_GO_RESOLUTION_DROPDOWN");
 	SQGSWindow.DefSQConst(engine, ScriptWindow::WID_GO_FULLSCREEN_BUTTON,                  "WID_GO_FULLSCREEN_BUTTON");
 	SQGSWindow.DefSQConst(engine, ScriptWindow::WID_GO_GUI_ZOOM_DROPDOWN,                  "WID_GO_GUI_ZOOM_DROPDOWN");
+	SQGSWindow.DefSQConst(engine, ScriptWindow::WID_GO_FONT_ZOOM_DROPDOWN,                  "WID_GO_FONT_ZOOM_DROPDOWN");
```

Comment:
extra space. You should be able to generate changes to this file by running `src/script/api/generate_widget.sh` (on Linux)

Reply:
That script doesn't seem to do anything for me (I'm running macOS) so I just removed the space manually.

Reply:
the script should work now